spec/requests/api/create_article_spec.rb

```diff
@@ -0,0 +1,50 @@
+RSpec.describe 'POST /api/articles', type: :request do
+  subject { response }
+  describe 'when the article is successfully created' do
```

Comment:
So it is easier to read specs you can put word successfully in the describe block. example:
```
describe 'successfully, when article is created'
```
So you now right away that is it happy path. Accordingly on sad path add 'unsuccessfully'

Reply:
Spec has been modified accordingly.